composer.json

```diff
@@ -28,7 +28,7 @@
   "prefer-stable": true,
   "require": {
     "oat-sa/generis": "15.11.0",
-    "oat-sa/tao-core": "48.66.1",
+    "oat-sa/tao-core": "dev-hotfix/adf-865/user-cannot-remove-properties as 48.66.1",
```

Comment:
Question: Shouldn't we make a new tag `48.66.2` in `tao-core` with the backport and point to that here?

Reply:
We will at the moment of the release. January was not released yet, so we do not create tags yet :)